packages/xchain-thorchain-query/src/utils/midgard.ts

```diff
@@ -45,6 +48,7 @@ export class Midgard {
     this.network = network
     this.config = config ?? defaultMidgardConfig[this.network]
     axiosRetry(axios, { retries: this.config.apiRetries, retryDelay: axiosRetry.exponentialDelay })
+    axios.defaults.headers.common['x-client-id'] = defaultMidgardConfig[this.network].customRequestHeaders
```

Comment:
Instead of using global `axios` defaults, which will be applied to EVERY requests, use custom instance defaults. See https://axios-http.com/docs/config_defaults

Reply:
oh okay good catch will fix this shortly

Reply:
removed